ui/lib/core/addon/components/list-view.hbs

```diff
@@ -2,8 +2,7 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: BUSL-1.1
 }}
-
-{{#if (or (and @items.meta @items.meta.total) @items.length)}}
+{{#if (or (and @items.meta @items.meta.filteredTotal) @items.length)}}
```

Comment:
Note: This change was required to show empty state on a filtered list. If the list is not filtered, then `meta.filteredTotal` is the same as `meta.total`, so this change does not break non-filtered functionality. 

I've manually tested this on existing pages that are using the `ListView` component, for example: 
* vault/secrets/kmip/scopes 
* vault/access